guide/english/bootstrap/cards/index.md

```diff
@@ -33,6 +33,23 @@ The structure of the card can be enhanced by the addition of a header and a foot
   <div class="card-footer">Footer content</div>
 </div>
 ```
+### Body
+--------
+The main container of Card is .card-body.
+
+###### This is what it would look like in your HTML document
+
+```html
+<div class="card">
+  <div class="card-body">
+    This is some text within a card body.
+  </div>
+</div>
+```
+###### Result
```

Comment:
This will only be useful if bootstrap is actually used on the final guide page.  Consider removing and replacing with an image or simply removing in another PR.